pilot/pkg/model/extensions.go

```diff
@@ -49,6 +49,10 @@ const (
 	WasmPolicyEnv = "ISTIO_META_WASM_IMAGE_PULL_POLICY"
 	// name of environment variable at Wasm VM, which will carry the resource version of WasmPlugin.
 	WasmResourceVersionEnv = "ISTIO_META_WASM_PLUGIN_RESOURCE_VERSION"
+
+	// WasmPluginResourceNamePrefix is the prefix of the resource name of WasmPlugin,
+	// preventing the name collision with other resources.
+	WasmPluginResourceNamePrefix = "istio.io/wasmplugin/"
```

Comment:
if we use `group/kind/namespace/name` it should be `extenstions.istio.io/Wasmplugin/`

Reply:
sounds good